.prettierignore

```diff
@@ -0,0 +1,35 @@
+node_modules
+.pnp
+.pnp.js
+coverage
+
+.changeset
+
+*.md
+*.mdx
+*.yml
+*.yaml
+
+.next/
+**/next-env.d.ts
+out/
+build
+
+.DS_Store
+*.pem
+
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+.pnpm-debug.log*
+
+.env
+.env.local
+.env.development.local
+.env.test.local
+.env.production.local
```

Comment:
Again, why? (These arent that necessary but still annoying when autosave doesnt work and you're left wondering why)

Reply:
I just copied over `.gitignore` which was previously being used as the `--ignore-file` for `format` in `package.json`